src/components/RepositorySearchCombobox.tsx

```diff
@@ -97,7 +97,7 @@ const RepositorySearchCombobox = ({
       position="relative"
       {...getComboboxProps()}
     >
-      <FormLabel {...getLabelProps()}>Repository</FormLabel>
+      <FormLabel {...getLabelProps()}>Type repository name</FormLabel>
```

Comment:
I'd suggest saying "enter" since you can paste the repo name for example.

```suggestion
      <FormLabel {...getLabelProps()}>Enter repository name</FormLabel>
```

Reply:
Let's go with `enter` then. 😉